docs/predictive-analytics/next-best-offer/source.json

```diff
@@ -0,0 +1,16 @@
+{
+  "connector.class"          : "MongoDbAtlasSource",
+  "name"                     : "recipe-mongodb-fleet_description",
+  "kafka.api.key"            : "<my-kafka-api-key>",
+  "kafka.api.secret"         : "<my-kafka-api-secret>",
+  "connection.host"          : "<database-host-address>",
+  "connection.user"          : "<database-username>",
+  "connection.password"      : "<database-password>",
+  "database"                 : "<database-name>",
+  "collection"               : "<database-collection-name>",
```

Comment:
@bbejeck I just looked into topic naming conventions for this connector (docs: https://docs.confluent.io/cloud/current/connectors/cc-mongo-db-source.html) and it seems the topic name generated will be of the form `<prefix>.<database-name>.<collection-name>`.  So as written today, if the user launches this recipe, the topic names won't match between what the connector generates and what the ksqlDB app calls the topics per the param `KAFKA_TOPIC`.  Can you reconcile this so the recipe _can_ work OOTB if the user is actually using this source connector?

Reply:
Updated to use the Postgres connector as I think that's a more realistic connector for this use case.